pyproject.toml

```diff
@@ -90,7 +90,8 @@ maintainer = [
   "mypy",
   "tbump",
   "autotyping",
-  "towncrier"
+  "towncrier",
+  "pytest-randomly"
```

Comment:
do we not need pytest-xdist?

Reply:
also, I believe we can remove towncrier now?

Reply:
we still use towncrier to add the dates on the changelog